src/pages/events/commands.md

```diff
@@ -119,12 +121,25 @@ You can also subscribe to a plugin event if it was registered in the `app/etc/co
 
 `--force`, `-f` Forces subscription to the event, even if it hasn't been defined locally.
 
+`--parent=<event code>` Specifies the name of the event to use as the basis of an event rule. If you specify this option, you must also specify the `--rules` option.
+
+`--rules=<field-name>|<operator>|<value>` Defines a rule that will be applied to the subscribed event. You can apply multiple rules to an event, but each rule must be defined separately. A rule definition must specify the field to be evaluated, an operator, and the value to be evaluated, in that order. The field name in a rule definition does not have to match a field specified with the `--fields` option.
+
 ### Example
 
+To subscribe to a native event:
+
 ```bash
 bin/magento events:subscribe observer.catalog_product_save_after --fields=sku --fields=stock_data.qty 
 ```
 
+To create and subscribe to a rule event based on `observer.catalog_product_save_after`:
+
+bin/magento events:subscribe observer.catalog_product_save_after_low_quantity \
```

Comment:
You need to wrap lines 138 - 141 in code fences. I would add a suggested edit, but GitHub misinterprets the suggested edit and renders a code block.